---
title: "Testing"
author: "Josh"
date: "2024-08-29"
output: html_document
---
this is a setup chuck which will allow me to not have to specify certain arugments through the other chunks 

```{r setup, echo=TRUE}
knitr::opts_chunk$set(echo = TRUE)
```

## R Markdown

####### bofa

# oiiiiiiiii

# my favorite websites
1. [iNaturalist](http://inaturalist.org)
2. [discogs](http://discogs.com)
3. [facebook marketplace](https://www.facebook.com/marketplace/?ref=app_tab)
<br>
<br>
![check out my indie music cassette collection](https://i.redd.it/avtnnqio03ld1.jpeg)
<br>
<br>

# y'all rock with tater tot hotdish???

<br>
<br>

![# this shit slaps yo](hotdish.jpeg)
<br>
<br>

## this is a script to prune VCF files to a certain number of SNPs. 
### I was using this to reduce my combined VCF files to make my genome datasets more manageable at later steps

```{bash, echo = TRUE, eval = FALSE}
# Convert the pruned VCF back to PLINK format
/programs/plink-1.9-x86_64-beta7/plink --vcf Primula_353super_LD_pruned_SNPs.recode.vcf --make-bed --allow-extra-chr --double-id --out Primula_genomeLD_pruned_thin

/programs/plink-1.9-x86_64-beta7/plink --bfile Primula_genomeLD_pruned_thin --thin-count 30000 --allow-extra-chr --double-id --recode --out Primula_genomeLD_pruned_thin_plink

/programs/plink-1.9-x86_64-beta7/plink --file Primula_genomeLD_pruned_thin_plink --allow-extra-chr --recode vcf --out Primula_genomeLD_pruned_thin_thinned


#gives missing proportion of loci for each individual
/local/workdir/jbl256/Installed_programs/vcftools-0.1.16/bin/vcftools --vcf Primula_genomeLD_pruned_thin_thinned.vcf --missing-indv

#average depth for each individual
/local/workdir/jbl256/Installed_programs/vcftools-0.1.16/bin/vcftools --vcf Primula_genomeLD_pruned_thin_thinned.vcf --depth 

#observed and expected heterozygosity
/local/workdir/jbl256/Installed_programs/vcftools-0.1.16/bin/vcftools --vcf Primula_genomeLD_pruned_thin_thinned.vcf --het


#create fasta file for downstream analyses
/local/workdir/jbl256/Installed_programs/vcf2phylip-2.8/vcf2phylip.py --input Primula_genomeLD_pruned_thin_thinned.vcf -f -n

#make structure file for later
/local/workdir/jbl256/Installed_programs/stacks-2.66/bin/populations -V Primula_genomeLD_pruned_thin_thinned.vcf -O ./ -M popmap.txt --threads 4 --ordered-export --structure

```

